doc/usage/extensions/autosummary.rst

```diff
@@ -74,13 +74,23 @@ The :mod:`sphinx.ext.autosummary` extension does this in two parts:
 
      The ``toctree`` option also signals to the :program:`sphinx-autogen` script
      that stub pages should be generated for the entries listed in this
-     directive.  The option accepts a directory name as an argument;
+     directive.
+
+     The option accepts a directory name as an argument;
      :program:`sphinx-autogen` will by default place its output in this
      directory. If no argument is given, output is placed in the same directory
-     as the file that contains the directive.
+     as the file that contains the directive.  The directory is interpreted as
+     relative to the directory of the source file, unless it begins with the
+     path separator ``/``, in which case it is relative to the documentation
+     root.  All paths should use the separator ``/`` regardless of the
+     operating system used for the build.
```

Comment:
Currently inconsistent, but I'd prefer to say "source directory" rather than "root", as there are at least 4 direcories that users may configure. 
```suggestion
     as the file that contains the directive.
     The directory is interpreted as relative to the directory of the source file,
     unless it begins with ``/``, in which case it is interpreted as
     relative to the documentation source directory.
```

Reply:
Done in e4dd14fa7